src/Vehicle/Vehicle.h

```diff
@@ -355,6 +355,9 @@ class Vehicle : public FactGroup
     /// Command vehicle to kill all motors no matter what state
     Q_INVOKABLE void emergencyStop(void);
 
+    /// Command vehicle to abort landing
+    Q_INVOKABLE void abortLanding(void);
```

Comment:
Can you change this to: ```abortLanding(double climbOutAltitude)``` and pass in the hardcoded value of 50 into it? That makes it more future ready.